package.json

```diff
@@ -1,27 +1,30 @@
 {
-  "author": "Zopim",
-  "name": "ipcluster",
-  "version": "0.0.12",
+  "author": "Zendesk",
+  "name": "@zendesk/ipcluster",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git@github.com:zopim/ipcluster.git"
+    "url": "git@github.com:zendesk/ipcluster.git"
+  },
+  "publishConfig": {
+    "registry": "https://zdrepo.jfrog.io/zdrepo/api/npm/npm"
   },
   "engines": {
     "node": ">=0.9.1"
   },
   "main": "lib/ipcluster.js",
   "dependencies": {
+    "lodash": "^4.17.15",
     "q": "1.5.1",
-    "lodash": "4.17.11",
     "toobusy-js": "0.4.3"
   },
   "devDependencies": {
-    "grunt": "0.4.5",
-    "grunt-cli": "",
-    "grunt-contrib-nodeunit": "*",
-    "grunt-contrib-jshint": "*"
+    "eslint": "^6.1.0",
+    "nodeunit": "^0.11.3"
   },
   "scripts": {
-    "test": "node_modules/grunt-cli/bin/grunt jshint"
+    "test": "nodeunit test/**/*.js",
```

Comment:
Can we run lint during CI? I think it would help catch style nits before PR review. Maybe add `pretest: npm run lint`?

Reply:
Never mind, lint is running in a separate Travis stage.